cmd/stack_removefromrepo.go

```diff
@@ -54,7 +54,7 @@ The updated repository index file is created in  ~/.appsody/stacks/dev.local dir
 				return errors.New("Required parameter missing. You must specify a repository name and a stack name")
 			}
 			if len(args) > 2 {
-				return errors.New("expected exactly 2 arguments: <repository> <stack>")
+				return errors.Errorf("Expected exactly 2 arguments: %v", cmd.Use)
```

Comment:
I would rather the word "two" than 2, which would be consistent with the other messages